le_utils/resources/presetlookup.json

```diff
@@ -57,7 +57,7 @@
 		"order": 4,
 		"kind": "video",
 		"allowed_formats": ["vtt"],
-		"convertible_formats": ["srt"]
+		"convertible_formats": ["srt", "ttml", "sami", "scc", "dfxp"]
```

Comment:
Plz remove `srt` from `convertible_formats` list --- I know it was there before you but not needed.